docker-app/qfieldcloud/core/admin.py

```diff
@@ -1025,17 +1026,20 @@ class OrganizationAdmin(QFieldCloudModelAdmin):
         "organization_owner__email__iexact",
     )
 
-    readonly_fields = (
-        "date_joined",
-        "storage_usage__field",
-    )
+    readonly_fields = ("date_joined", "storage_usage__field", "active_users")
 
     list_select_related = ("organization_owner",)
 
     list_filter = ("date_joined",)
 
     autocomplete_fields = ("organization_owner",)
 
+    @admin.display(description=_("Active users (last 90 days)"))
+    def active_users(self, instance) -> int:
+        now = datetime.now()
+        three_months_ago = now - timedelta(days=90)
+        return len(list(instance.active_users(now, three_months_ago)))
+
```

Comment:
Actually you can use `org.current_subscription.active_users_count`, which will do it for the last billing period, if the subscription was active and probably raise? otherwise. If the latter case we better return `None`, which makes the admin show `-`.

Reply:
And one remark, look at the page performance. If it get's 100%+ slower, probably it should not be a column in the List view, but readonly field in the edit view.